packages/grafana-data/src/types/dataFrame.ts

```diff
@@ -136,6 +136,7 @@ export interface Field<T = any, V = Vector<T>> {
    */
   config: FieldConfig;
   values: V; // The raw field values
+  additionalValues?: Record<string, V>;
```

Comment:
Given that this is a pattern we know applies to time, and struggle to think of good cases for other things should we do something liket:
```suggestion
  /** When type === FieldType.time, we can support nanosecond time bla bla bla... 8?
  nanos?: number[];
```

Reply:
can do 👍 .. just to doublecheck, we can go directly to a javascript-array, no need to have an `ArrayVector` ?